etc/xivo-build-tools/packages

```diff
@@ -106,10 +96,7 @@ xivo-lib-rest-client
 xivo-manage-db
 xivo-monitoring
 xivo-python-consul-packaging
-xivo-python-flask-cors-packaging
 xivo-python-flask-menu-packaging
-xivo-python-flask-wtf-packaging
```

Comment:
Why removing flask-wtf? It is not provided by debian and still use for wazo-ui

Reply:
The debian name is not the same: https://packages.debian.org/buster/python3-flaskext.wtf